packages/calcite-components/src/components/functional/XButton.tsx

```diff
@@ -19,7 +20,6 @@ export function XButton({
   label,
   onClick,
   ref,
-  scale,
```

Comment:
I totally missed this in the review. You'll need to pass the `scale` prop directly since you can't use `this` reliably here. Sorry about that! 😅